recipes/vtkbool/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set version = "3.0rc1" %}
+{% set vtkbool_version = "3.0-rc1" %}
+
+package:
+  name: vtkbool
+  version: {{ version }}
+
+source:
+  url: https://github.com/zippy84/vtkbool/archive/refs/tags/v{{ vtkbool_version }}.tar.gz
+  sha256: a422662a7679b6f7e433596c4f80e1569e2b1c55b6a277c008881cd3ca4f7e1a
+  patches:
+    - modify-cmakelists.patch  # patch cmakelists to support building in conda env
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - cmake
+    - {{ compiler('cxx') }}
+    - ninja
+  host:
+    - python
+    - vtk
+  run:
+    - python
+    - vtk
+
+test:
+  imports:
+    - vtkbool
```

```suggestion
test:
  commmands:
    - test -f $PREFIX/include/vtkBoolModule.h  # [unix]
    - test -f $PREFIX/lib/vtkBool${SHLIB_EXT}  # [unix]
  imports:
    - vtkbool
```
https://conda-forge.org/docs/maintainer/adding_pkgs.html#simple-existence-tests

Add tests which indicate which types of artifacts are packaged.